tests/markFeatureWriter_test.py

```diff
@@ -16,9 +16,10 @@ def test_add_classes(self):
         glyph.appendAnchor(glyph.anchorClass(
             anchorDict={'name': '_bottom', 'x': 100, 'y': 0}))
         lines = []
-        writer = MarkFeatureWriter(
-            ufo, anchorList=(('bottom', '_bottom'),), mkmkAnchorList=(),
-            ligaAnchorList=((('top_1', 'top_2'), '_top'),))
+        writer = MarkFeatureWriter(ufo)
+        writer.anchorList = (('bottom', '_bottom'),)
+        writer.mkmkAnchorList = ()
+        writer.ligaAnchorList = ((('top_1', 'top_2'), '_top'),)
```

Comment:
I think subclassing the MarkFeatureWriter is a cleaner solution if one wants to add custom rules for anchor naming, as that's where it belongs to. The FeaturesCompiler should not know about the details of any of the specific features; that's what the specific feature writers are for.